packages/core/parcel/src/cli.js

```diff
@@ -340,6 +348,10 @@ async function normalizeOptions(command): Promise<InitialParcelOptions> {
       INTERNAL_ORIGINAL_CONSOLE.warn(
         chalk.bold.yellowBright(`⚠️  Port ${command.port} could not be used.`),
       );
+      // If port was provided using -p or --port flag then throw an Error
+      if (isCustomPort && nodeEnv === 'development') {
```

Comment:
Is the development check neccessary?

Reply:
I checked and I think it wasn't - thanks